_layouts/default.html

```diff
@@ -135,7 +135,6 @@
             <a class="nav--v__link--footer" href="https://rubygems.org/stats">Stats</a>
             <a class="nav--v__link--footer" href="http://blog.rubygems.org/">Blog</a>
             <a class="nav--v__link--footer" href="https://rubygems.org/pages/about">About</a>
-            <a class="nav--v__link--footer" href="http://help.rubygems.org/">Help</a>
```

Comment:
Thank you for updating these. Instead of removing these can we please update these to mailto links?
```
<a href="mailto:support@rubygems.org"..
```
for markdown:
```
[text](mailto:support@rubygems.org)
```

Reply:
Okay, updated.